DFC.App.ContactUs/appsettings-template.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug"
+    }
+  },
+  "AllowedHosts": "*",
+  "Configuration": {
+    "CosmosDbConnections": {
+      "ContentPages": {
+        "AccessKey": "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==",
```

Comment:
remove app key

Reply:
This is the well-known public key for the Cosmos Emulator - not a security issue